front/pages/poke/[wId]/data_sources/[name]/index.tsx

```diff
@@ -710,4 +741,32 @@ function NotionUrlCheckOrFind({ owner }: { owner: WorkspaceType }) {
   );
 }
 
+function SlackWhitelistBot({ owner }: { owner: WorkspaceType }) {
+  const [botName, setBotName] = useState("");
+
+  return (
+    <div className="mb-2 flex flex-col gap-2 rounded-md border px-2 py-2 text-sm text-gray-600">
+      <div className="flex items-center gap-2">
+        <div>Whitelist slack bot or workflow</div>
+        <div className="grow">
```

Comment:
Can we show the list of whitelisted bots next to the input 🙏

Reply:
That would need adding a custom connectors api endpoint, a bit overkill at the time?
80/20'd it by adding a metabase link to display them